src/ext/java/org/opentripplanner/ext/transmodelapi/model/util/EncodedPolylineBeanWithStops.java

```diff
@@ -0,0 +1,35 @@
+package org.opentripplanner.ext.transmodelapi.model.util;
+
+import org.opentripplanner.model.StopLocation;
+import org.opentripplanner.util.model.EncodedPolyline;
+
+public class EncodedPolylineBeanWithStops {
```

Comment:
This could probably be a record.

Reply:
Indeed. It's not a hard rule but I think we want to use immutable objects with final fields as much as possible.